components/shared-resources/globally-shared-secrets-clusterrole.yaml

```diff
@@ -11,9 +11,6 @@ rules:
       - sharedsecrets
     resourceNames:
       - redhat-appstudio-user-workload
-      - snyk-shared-secret
```

Comment:
remind me, what's the state of the snyk usage?

Reply:
currently pipelines are having parameter which expect secret name - https://github.com/redhat-appstudio/build-definitions/blob/main/pipelines/template-build/template-build.yaml#L49 
It's set for our pipelineruns in tekton-ci namespace. So used by our services and optional for customers (must set it).

Reply:
HACBS-1778